bindings/csharp/Ckzg.Test/ReferenceTests.cs

```diff
@@ -354,4 +360,158 @@ public void TestVerifyBlobKzgProofBatch()
     }
 
     #endregion
+
+    #region ComputeCellsAndKzgProofs
+
+    private class ComputeCellsAndKzgProofsInput
+    {
+        public string Blob { get; set; } = null!;
+    }
+
+    private class ComputeCellsAndKzgProofsTest
+    {
+        public ComputeCellsAndKzgProofsInput Input { get; set; } = null!;
+        public List<List<string>>? Output { get; set; } = null!;
+    }
+
+    [TestCase]
+    public void TestComputeCellsAndKzgProofs()
+    {
+        Matcher matcher = new();
+        matcher.AddIncludePatterns(new[] { "*/*/data.yaml" });
+
+        IEnumerable<string> testFiles = matcher.GetResultsInFullPath(_computeCellsAndKzgProofsTests);
+        Assert.That(testFiles.Count(), Is.GreaterThan(0));
+
+        foreach (string testFile in testFiles)
+        {
+            string yaml = File.ReadAllText(testFile);
```

Comment:
In general cases could be separated to a source passed to TestCaseSource attribute

Reply:
Good idea. I only did this for the new ones. Let's follow-up with a PR for the old tests later.